### PR TITLE
add sample mode

### PR DESCRIPTION
**Pull request by zhouzilong2020**
Added sample mode for satrting the frontend.
Main change: Import json file dynamically according to the given mode.

PTAL at the README file @tianzhou

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,23 +38,33 @@ The market lacks a tool for developers to compare different database products be
 
 This project is under development and is very unstable. The way to start this project may improve as process goes on.
 
-### Installing Frontend Dependencies
-```
-cd ./frontend && pnpm i
-```
-
 ### Fetching Data
+Sample data is provided at [here](https://github.com/bytebase/dbcost/blob/main/store/data/sample.json). Your may start dbcost in sample mode, without fetching your own data. If you decide to start with sample data, just skip this step.
```
Comment (tianzhou) [inline README.md:42]:
```suggestion
Sample data is provided at [here](https://github.com/bytebase/dbcost/blob/main/store/data/sample.json). You may start dbcost in sample mode without fetching the cloud provider's latest pricing data. If you decide to start with sample data, just skip this step.
```

File: README.md [hunk 2]
```diff
@@ -37,23 +38,33 @@ The market lacks a tool for developers to compare different database products be
...
+
+Or, if you would like to fetch the newest data online, please apply for a [GCP API KEY](https://cloud.google.com/apigee/docs/api-platform/security/api-keys?hl=en&_ga=2.248673159.1879775959.1652425553-1923221278.1651636210&_gac=1.19919946.1652425553.Cj0KCQjwg_iTBhDrARIsAD3Ib5jxpCArIqLqlXF61hULy8r46r2GwzWKde-4nLdzetVquoP6ed1R_QsaAjXVEALw_wcB) with access to the [Cloud Billing API](https://cloud.google.com/billing/docs/reference/rest) yourself first.
```
Comment (tianzhou) [inline README.md:44]:
```suggestion
Or, if you would like to fetch the newest data online, please apply for a [GCP API KEY](https://cloud.google.com/apigee/docs/api-platform/security/api-keys) with access to the [Cloud Billing API](https://cloud.google.com/billing/docs/reference/rest) yourself first. 
```

Comment (tianzhou) [reply README.md:44]: Please list instructions for each cloud provider respectively. If AWS doesn't need the key, we also mention it explicitly.

File: frontend/src/stores/dbInstance.ts [hunk 3]
```diff
@@ -65,7 +65,19 @@ export const useDBInstanceStore = defineStore("dbInstance", {
 
   actions: {
     loadDBInstanceList() {
-      const receiver = [];
+      let sourceFileName = "rds.json";
```
Comment (tianzhou) [inline frontend/src/stores/dbInstance.ts:68]: rds is AWS specific. GCP is called cloud sql. so we need a more generic name

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by zhouzilong2020** [comment]
PTAL

**Pull request merged**